libcontainer/specconv/spec_linux.go

```diff
@@ -710,7 +711,7 @@ func setupSeccomp(config *specs.LinuxSeccomp) (*configs.Seccomp, error) {
 		return nil, nil
 	}
 
-	// No default action specified, no syscalls listed, assume seccomp disabled
+	// No default action specified, no unixs listed, assume seccomp disabled
```

Comment:
Oops. Yep. I thought I caught all the spots I needed to keep syscalls. I'll take a closer look.